src/Stores/Promises/reducer.js

```diff
@@ -0,0 +1,29 @@
+/* @flow */
+import type { PromiseType, Action } from '../../types';
+
+export type State = {
+  +promises: ?Array<PromiseType>,
+  +loading: boolean,
+  +error: *,
+};
+
+const initialState = {
+  promises: null,
+  loading: false,
+  error: null,
+};
+
+const promisesReducer = (state: State = initialState, action: Action): State => {
+  switch (action.type) {
+    case 'PROMISES_LOAD':
+      return { ...state, loading: true, promises: null };
```

Comment:
Maybe set error to null in load?

If it goes:
Load
Error
Load
Success

Then error will still have the action.error value from the error state in the success state